doc/src/manual/functions.md

```diff
@@ -331,12 +331,36 @@ Named tuples are very similar to tuples, except that fields can additionally be
 using dot syntax (`x.a`) in addition to the regular indexing syntax
 (`x[1]`).
 
-## Multiple Return Values
+## Destructuring Assignment and Multiple Return Values
 
-In Julia, one returns a tuple of values to simulate returning multiple values. However, tuples
-can be created and destructured without needing parentheses, thereby providing an illusion that
-multiple values are being returned, rather than a single tuple value. For example, the following
-function returns a pair of values:
+A comma-separated list of variables (optionally wrapped in parentheses) can appear on the
+left side of an assignment: the value on the right side is _destructured_ by iterating
+over and assigning to each variable in turn:
+
+```jldoctest
+julia> (a,b,c) = 1:3
+1:3
+
+julia> b
+2
+```
+
+The value on the right should be an iterator (see [Iteration interface](@ref man-interface-iteration))
+at least as long as the number of variables on the left (any excess elements of the
+iterator are ignored). Additionally a single underscore `_` (which is an otherwise invalid
+variable name, see [Allowed Variable Names](@ref man-allowed-variable-names)) can be used
+on the left to avoid assigning specific elements:
+
+```jldoctest
+julia> _, _, _, d = 1:10
+1:10
+
+julia> d
+4
+```
+
+This can be used to simulate returning multiple values from functions by returning a tuple or
+other iterable value. For example, the following function returns a two values:
```

Comment:
```suggestion
This can be used to return multiple values from functions by returning a tuple or
other iterable value. For example, the following function returns two values:
```